src/App.js

```diff
@@ -30,14 +30,51 @@ const App = () => {
   // This starts state off as a 2D array of JS objects with
   // empty value and unique ids.
   const [squares, setSquares] = useState(generateSquares());
-
+  const [currentPlayer, setCurrentPlayer] = useState(PLAYER_1);
+  const [winner, setWinner] = useState('');
+ 
+  //const winner = checkForWinner(squares);
   // Wave 2
   // You will need to create a method to change the square 
   //   When it is clicked on.
   //   Then pass it into the squares as a callback
+  const onClickCallback = (id) => {
```

Comment:
Just like you are setting squares with new squares and setting the current player to the updated player, you want to check for the winner on the onClickCallBack. Add `setWinner(checkForWinner());` in this function